css/index.css

```diff
@@ -1 +1,604 @@
+  margin-bottom: 15px;
+}
+
+/* --------------------------------------------------------- */
+/* --------------------------------------------------------- */
+/* --------------------------------------------------------- */
+/* ---------------------- Main Page CSS -------------------- */
+/* --------------------------------------------------------- */
+/* --------------------------------------------------------- */
+/* --------------------------------------------------------- */
+
+.container {
+  max-width: 880px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+/* Header-related CSS */
+
+header {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  width: 100%;
+  align-items: center;
+  vertical-align: top;
+  margin-top: 35px;
+}
+
+header nav { 
+  width: 70%;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+}
+
+header nav a {
+  display: inline-block;
+  font-size: 1.4rem;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+  
+/* CTA-related CSS */
+
+.cta {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 69px;
+}
+
+.cta .cta-text {
+  align-self: center;
+  margin-top: 30px;
+  font-size: 7.2rem;
+  text-align: center;
+  padding: 0px 8.5%;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 10px;
+  padding: 6px 40px;
+  font-size: 1.7rem;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta-img {
+  right: 0px;
+  width: 100%;
+  height: auto;
+  border-radius: 5%;
+  margin-bottom: 50px;
+}
+
+.cta img:hover {
+  transform: rotate(360deg);
+  transition: 1.2s;
+}
+
+/* Main Content */
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+}
+
+.main-content .top-content .text-content {
+  font-size: 1.6rem;
+  width: 49%;
+  display: inline-block;
+}
+
+.main-content .middle-img {
+  margin-top: 30px;
+  width: 100%;
+  height: auto;
+  transition: .3s;
+}
+
+.main-content .middle-img:hover {
+  transform: scale(1.1);
+  transition: .3s;
+}
+
+.main-content .bottom-content {
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  margin-top: 37px;
+}
+
+.main-content .bottom-content .text-content {
+  font-size: 1.6rem;
+  width: 33%;
+  display: inline-block;
+}
+
+.bottom-content {
+  padding-bottom: 40px;
+  border-bottom: 1px solid black;
+}
+
+/* Contact & Footer CSS */
+
+.contact {
+  padding: 39px 0 0;
+  font-size: 1.6rem;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 1.4rem;
+}
+
+/* Main page CSS for screens 800px and below */
+@media (max-width: 800px) {
+
+  .container {
+    width: 92%;
+    max-width: 500px;
```

Comment:
Why is the max-width 500px here?

Reply:
I think I did that to make the gutters a fixed size, but I see now why I didn't necessarily need it.